docs/basics/101-139-gin.rst

```diff
@@ -0,0 +1,214 @@
+.. _gin:
+
+Dataset hosting on GIN
+----------------------
+
+`GIN <https://gin.g-node.org/G-Node/Info/wiki>`__ (G-Node infrastructure) is a
+free data management system designed for comprehensive and reproducible management
+of scientific data. It is a secure, web-based repository store and provides
+fine-grained access control to share data. :term:`GIN` builds up on :term:`Git` and
```

Comment:
This sounds like we have certified that. Either remove the "secure" claim, or state "it is advertised as". I prefer the former.